flink-libraries/flink-cep/src/test/java/org/apache/flink/cep/operator/CepRuntimeContextTest.java

```diff
@@ -110,38 +111,41 @@ public void testCepRuntimeContext() {
         final String taskName = "foobarTask";
         final OperatorMetricGroup metricGroup =
                 UnregisteredMetricsGroup.createOperatorMetricGroup();
-        final int numberOfParallelSubtasks = 42;
-        final int indexOfSubtask = 43;
+        final int numberOfParallelSubtasks = 43;
+        final int indexOfSubtask = 42;
         final int attemptNumber = 1337;
-        final String taskNameWithSubtask = "barfoo";
+        final String taskNameWithSubtask = "foobarTask (43/43)#1337";
```

Comment:
How are these changes related to this commit?

Reply:
These tests are conflicted with the newly added correctness check in the `TaskInfoImpl`. For example, the subtask ID shouldn't be large than or equal to the operator parallelism number. I've fixed these errors in the commit `[hotfix] Fix the logical errors for the tests related to task metadata`.